lib/src/test/kotlin/net/dean/jraw/test/unit/GildingsTest.kt

```diff
@@ -0,0 +1,267 @@
+package net.dean.jraw.test.unit
+
+import com.winterbe.expekt.should
+import net.dean.jraw.JrawUtils
+import net.dean.jraw.models.Comment
+import net.dean.jraw.models.Submission
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.describe
+import org.jetbrains.spek.api.dsl.it
+
+class GildingsTest : Spek({
+
+    describe("test new submission gildings, they should not be null") {
+
+        val json = "{" +
```

Comment:
Request JSON from the reddit API via HTTP in-test. This should be done in a before each block or if this is the only test that uses this data, inside the test itself.

Reply:
I see that it can be done either with a travisCI config or with a credentials.json file.
I tried the second option but I had an error with the http request, I didn't waste anymore time.
Maybe I had a mistake in my json file, do you have a template that I can use?

Reply:
The rest is already done

Reply:
https://github.com/mattbdean/JRAW#contributing

Reply:
Ok done